mlflow/keras/__init__.py

```diff
@@ -1,24 +1,45 @@
 # MLflow Keras 3 flavor.
+import keras
+from packaging.version import Version
 
-from mlflow.keras.autolog import autolog
-from mlflow.keras.callback import MLflowCallback
-from mlflow.keras.load import _load_pyfunc, load_model
-from mlflow.keras.save import (
-    get_default_conda_env,
-    get_default_pip_requirements,
-    log_model,
-    save_model,
-)
+if Version(keras.__version__) < Version("3.0.0"):
+    from mlflow.tensorflow import (
```

Comment:
undo the deletion from the original PR, keeping it behind a version check instead:

https://github.com/mlflow/mlflow/pull/10830/files#diff-6e6f5be585845fe6e5d51023750107215d9d70d0f5039ad0a5c8a4b570c741d8L1-L9

Reply:
cc @chenmoneygithub does this look alright to you?

Reply:
sounds good! is it cool if i merge this first to unblock x-version tests? once you get around to making the difference in the transformers flavor, you can delete this here.